File: beginner/pagina12/3076/3076.php
<?php 

    while($ano = (int) readline()){

        if($ano <= 100){
            echo "1\n";
        } else {
            $seculo = (int) $ano / 100;
            if($ano % 100 != 0){
                $seculo += 1;
            }

            echo "$seculo\n";
        }
    }

?>

File: beginner/pagina12/3076/3076.rs
use std::io;

fn main() {

    let mut input = String::new();

    while io::stdin().read_line(&mut input).unwrap() > 0 {

        let ano: u32 = input.trim().parse().unwrap();

        let seculo = if ano % 100 == 0 { ano / 100 } else { ano / 100 + 1 };

        println!("{}", seculo);

        input.clear();
    } 
}